packages/blade/src/components/Breadcrumb/_decisions/decisions.md

```diff
@@ -0,0 +1,203 @@
+# Breadcrumb Decisions <!-- omit in toc -->
+
+Breadcrumbs are used for navigating through or to show user’s location in an application.
+
+<img width="426" alt="Breadcrumb Thumbnail" src="./breadcrumb-thumbnail.png" />
+
+- [Design](#design)
+- [Anatomy](#anatomy)
+- [Basic Usage](#basic-usage)
+- [API](#api)
+  - [Breadcrumb](#breadcrumb)
+  - [BreadcrumbItem](#breadcrumbitem)
+  - [Examples](#examples)
+    - [Show Last Separator](#show-last-separator)
+  - [Usage with Routing Libraries](#usage-with-routing-libraries)
+- [Accessibility](#accessibility)
+- [Open Questions](#open-questions)
+- [References](#references)
+
+## Design
+
+[Figma Link](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade-DSL?type=design&node-id=81170-57117&mode=design&t=qeUPSKFBYZfBCFBT-0) to all variants of the Breadcrumb component
```

Comment:
Maybe use [this link](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade-DSL?type=design&node-id=81454%3A67196&mode=design&t=DhLqf1C1xX8lcmQc-1)